core/objects.go

```diff
@@ -483,6 +483,12 @@ type SuggestedWindow struct {
 	End   time.Time `json:"end"`
 }
 
+// IsWithin returns true if the given time is within the suggested window,
```

Comment:
Could you add a test to `TestARI` that uses `IsWithin` please?

Reply:
Ooof, this is turning out to be a little bit tougher than I had originally bargained for. I'm going to put a little more thought into it though.

Reply:
I decided to just unit test IsWithin().